docs/units-and-global-variables.rst

```diff
@@ -232,7 +232,9 @@ Mathematical and Cryptographic Functions
     the ecrecover function remained unchanged.
 
     This is usually not a problem unless you require signatures to be unique or
-    use them to identify items. OpenZeppelin have a `ECDSA helper library <https://docs.openzeppelin.com/contracts/2.x/api/cryptography#ECDSA>`_ that you can use as a wrapper for ``ecrecover`` without this issue.
+    use them to identify items.
+    
+    OpenZeppelin have a `ECDSA helper library <https://docs.openzeppelin.com/contracts/4.x/api/utils#ECDSA>`_ that you can use as a wrapper for ``ecrecover``, but BE CAREFUL which helper you pick - both `tryRecover(hash, signature)` and `recover(hash, signature)` accept signatures in two forms (65 and 64 bytes), thus are not fixing anything. Use helpers which accept `r`, `v`, `s` arguments instead.
```

Comment:
I agree with the suggestion, but the current wording sounds rather confusing to me. "Why are they not fixing anything", a reader will likely ask. Is there an external link from OZ explaining those things that we can just link here?

Reply:
Actually the fixed it ~2 weeks ago, the bug was present for 1 year and 4 months there. See my second comment: https://github.com/ethereum/solidity/pull/13419#issuecomment-1221418771

So i'm wondering if we should drop this PR or just warn users about the problem in older versions.

Reply:
We could keep the part of the PR that updates the OZ version in the link but remove the text part. Wdyt?

Reply:
Why not just remove the mention of OpenZeppelin?

Reply:
Whilst it's a bit weird having our docs point to a third party library, I don't think it's necessarily bad that we're mentioning OZ, since they're a major part of the ecosystem, so I'd side with @leonardoalt; simply update the OZ link to point to a later version, and remote the rest.